2.2.3/bullseye/include/entrypoint

```diff
@@ -1,5 +1,55 @@
 #!/usr/bin/env bash
 set -e
+set -x
+
+
+# Optional sidecar logging keepalive support
+if [[ ! -z ${SIDECAR_LOGGING_KEEPALIVE_PORT+x} ]] && [[ "$COMPONENT" =~ ^(scheduler|webserver|worker)$ ]] && [[ $1 != "python" ]]; then
+  SIDECAR_LOGGING_KEEPALIVE_SOCKET=/dev/tcp/127.0.0.1/${SIDECAR_LOGGING_KEEPALIVE_PORT}
+  echo "Connecting to sidecar logging keepalive-daemon on ${SIDECAR_LOGGING_KEEPALIVE_SOCKET}"
+  # loop while we wait for the socket to come up, quashing stderr in the process
+  # once we do manage to make a connection, redirect it to fd #7
+  while true; do
+    set +e
+    exec 4>&2 2>/dev/null 7<>${SIDECAR_LOGGING_KEEPALIVE_SOCKET} 2>&4
+    set -e
+    if [ $? -eq 0 ]; then
+        echo "Connected to sidecar logging keepalive-daemon on ${SIDECAR_LOGGING_KEEPALIVE_SOCKET}"
+	break;
+    else
+      sleep .05;
+    fi
+  done
+fi
+
+# Optional support for shipping unstructured logs
+if [[ ! -z ${UNSTRUCTURED_LOGGING_HOST+x} ]] && [[ "$COMPONENT" =~ ^(scheduler|webserver|worker)$ ]] && [[ $1 != "python" ]]; then
+  if [[ -z ${COMPONENT} ]]; then
+    echo "COMPONENT must be set when using sidecar logging."
+    exit 2;
+  fi
+  UNSTRUCTURED_LOGGING_SOCKET=/dev/tcp/${UNSTRUCTURED_LOGGING_HOST}/${UNSTRUCTURED_LOGGING_PORT}
+  echo "Connecting to tcp logging daemon on ${UNSTRUCTURED_LOGGING_SOCKET}"
+  while true; do
+    set +e
+    # create a pointer to the origianl stdout and stderr
+    exec 3>&1 4>&2 2>&4
+    # clobber old stdout and stderr
```

Comment:
Is this supposed to have a line with some redirects below it?

Reply:
I think the connection happens on 36. If it was successful it lets the redirect stand.

Reply:
What i mean is there's a comment that says "Clobber old stdout and stderr" but there is nothing below that comment.